### ISSUE TITLE
--> and ==> don't quite act the same way

### ISSUE THREAD

**Issue by marnen** [marnen/borg-sans-mono#6]
Probably not really a problem in practice.

**Comment by marnen** [comment]
Hmm. Maybe `-->` should just not ligate. Is there any language that actually uses it as an arrow? (The only use I can think of for it is as a HTML/XML comment delimiter.)

**Comment by HonoluluHenk** [comment]
Xml uses  as comment delimiters (both look very strange with borg-sans-mono)
Would be nice to have these as ligatures (see e.g. Fira Code)

**Comment by marnen** [comment]
@HonoluluHenk I agree, these look kind of strange in the current version of the font. Do you have any thoughts on what a good --> ligature might look like?
